### DISCUSSION TITLE
Add Printing support?

### DISCUSSION THREAD

**Discussion by galgogergo** [pdfarranger/pdfarranger#793] [Q&A] [answered]
Hi,

I use PDF Arranger a LOT during my work hours, and like how easily you can manipulate pages in a PDF (adding/removing/extracting/exporting/rotating and saving). 
The only feature I'm missing is printing.

Would it be really difficult to add printing support? If even with the default printing panel which the OS supports. (not sure if it exists in Windows, or you have to always construct your own when coding. On Mac there's definitely an option to "Print using system dialog.")

So, add printing, basically :)

**Comment by dreua** [comment]
Pretty sure we already have an issue for that, please use the search featured.

**Comment by jeromerobert** [comment, marked as answer]
This was implemented in version 1.9.0.

**Reply by galgogergo** [reply]
Thanks, I've updated since.

**Comment by galgogergo** [comment]
I found a bug, though!
Sorry, I'm new to GitHub, would you point me to where it should be posted? Or should I just paste it here?

Thanks, 
G

**Comment by tactilis** [comment]
> would you point me to where it should be posted?

[https://github.com/pdfarranger/pdfarranger/issues](https://github.com/pdfarranger/pdfarranger/issues)

Click the `New Issue` button

**Reply by galgogergo** [reply]
thanks, done!
https://github.com/pdfarranger/pdfarranger/issues/806
